experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -2615,6 +2596,39 @@ def test_get_kpi_metrics_returns_correct_metrics(
 
         self.assertListEqual(kpi_metrics, expected_kpi_metrics)
 
+    def test_get_defaults_metrics_with_exclusions(self):
+        experiment = NimbusExperimentFactory.create()
+        branch_a = NimbusBranchFactory.create(
+            experiment=experiment, name="Branch A", slug="branch-a"
+        )
```

Comment:
Is this needed for the test to work? Not a problem exactly just seems unnecessary for what's being tested here, unless it's necessary for the experiment to be valid, or something similar.